main/main.gd

```diff
@@ -55,6 +54,14 @@ func start_game(num_rounds, num_players):
 				break
 			await player_turn(player)
 
+	State.players.sort_custom(func(player1, player2): 
+			return player1.riesen + player1.riesen < player2.riesen + player2.patente
```

Comment:
The calculation is off, as the first sum is not the same as the second one. Also, does a patent have the same value as the money currency?

Reply:
As far as i understand it, the value is the same. Half of the value is simply lost when buying patente

Reply:
Oh a typo

Reply:
Changed it to
`State.players.sort_custom(func(player1, player2): 
			return player1.riesen + player1.patente < player2.riesen + player2.patente
	)
`